docs/src/main/paradox/common/binary-compatibility-rules.md

```diff
@@ -78,16 +74,16 @@ If you accidentally mix Pekko versions, for example through transitive
 dependencies, you might get a warning at run time such as:
 
 ```
-You are using version 2.6.6 of Pekko, but it appears you (perhaps indirectly) also depend on older versions 
-of related artifacts. You can solve this by adding an explicit dependency on version 2.6.6 of the 
+You are using version 1.0.6 of Pekko, but it appears you (perhaps indirectly) also depend on older versions 
+of related artifacts. You can solve this by adding an explicit dependency on version 1.0.6 of the 
 [pekko-persistence-query] artifacts to your project. Here's a complete collection of detected 
-artifacts: (2.5.3, [pekko-persistence-query]), (2.6.6, [pekko-actor, pekko-cluster]).
+artifacts: (1.0.3, [pekko-persistence-query]), (1.0.6, [pekko-actor, pekko-cluster]).
 See also: https://pekko.apache.org/docs/pekko/current/common/binary-compatibility-rules.html#mixed-versioning-is-not-allowed
 ```
 
 The fix is typically to pick the highest Pekko version, and add explicit
 dependencies to your project as needed. For example, in the example above
-you might want to add `pekko-persistence-query` dependency for 2.6.6.
+you might want to add `pekko-persistence-query` dependency for 1.0.6.
```

Comment:
I'll re-review all these versions later. It's hard to come up with good values when we don't have any releases. I'm almost leaning towards removing the examples and maybe even just mentioning that we will likely stick closely to established norms used for Akka releases and link to their equivalent page. The version numbers are more understandable when you have a history of releases to reason about.

Reply:
> The version numbers are more understandable when you have a history of releases to reason about.

Ah I think I understand what the paragraph is trying to say know, i.e. its meant to be hypothetical to illustrate versioning but as you said its somewhat confusing since `1.0.6` does't even exist right now.

In any case I will approve the PR and leave it to your judgement.